gensrc/proto/internal_service.proto

```diff
@@ -456,14 +457,19 @@ message PResetRPCChannelResponse {
     repeated string channels = 2;
 };
 
+message PEchoRequest {};
```

Comment:
why named PEchoRequest ？ what dose echo mean

Reply:
In the official example of brpc, use EchoRequest to represent empty Request.

Reply:
brpc called EchoRequest because of it's service called `echo`
 you can use `PEmptyRequest`  or `PEmptyTransmitDataParams`